### PR TITLE
Enums

### PR DESCRIPTION
**Pull request by guillaumemoby**
As @tonnenpinguin suggested, I have added the enums.
I don't know if we should split the drills between the frisbee and fitness.

Maybe this difference should more appears on the drillpage.js and the Drilllistpage.js

### CODE REVIEW

File: src/Fixtures/index.js [hunk 1]
```diff
@@ -1,27 +1,66 @@
...
-  /*   const types: {
-    FITNESS: 'fitness',
-    TECHNICAL: 'technical',
-  }, */
+export const types = {
+  FITNESS: 'fitness',
+  TECHNICAL: 'technical',
+};
+
+export const intensity = {
```
Comment (sallesma) [inline src/Fixtures/index.js]: I think enum names should be plural, like you did for types

Comment (guillaumemoby) [reply src/Fixtures/index.js:9]: You are right. Thanks.
I want to upload only the file index.js, is there a easy way to only add this one?

**Comment by arribbar** [comment]
@guillaumemoby 
You have three things to do before we can merge this PR:
- Update your branch with master. When you create a new branch, always create it when you are on master and this one is up to date
- Resolve conflicts. You can see when you do your PR that you have conflicts with master. It means we cannot merge it. To resolve conflicts, from enums branch, you need to do a `git pull origin master` to get last commits from master. Then, resolve conflicts, commit these changes and update your branch on Github
- Isolate your last commit from the previous ones

I can help you or you ask to @sallesma

**Comment by tonnenpinguin** [comment]
The easiest way might be to create a new branch from master and cherry pick the commit with your changes. Then submit the new branch and open a new PR.

File: src/Fixtures/index.js [hunk 2]
```diff
@@ -1,11 +1,892 @@
...
+
+export const seasonTimings = {
+  OFFSEASON: 'Off-season',
+  PRESEASON: 'Pre-season',
+  INSEASON: 'In-season',
+  ANYTIME: 'Anytime',
+};
+
+export const equipmentLabels = {
+  NOEQUIPMENT: 'No Equipment',
```
Comment (tonnenpinguin) [inline src/Fixtures/index.js:43]: use underscores for better readability

File: src/Fixtures/index.js [hunk 3]
```diff
@@ -1,11 +1,892 @@
...
+};
+
+export const seasonTimings = {
+  OFFSEASON: 'Off-season',
+  PRESEASON: 'Pre-season',
+  INSEASON: 'In-season',
+  ANYTIME: 'Anytime',
+};
+
+export const equipmentLabels = {
```
Comment (tonnenpinguin) [inline src/Fixtures/index.js:42]: I would actually upcase the names to make it clearer when the code code is called (--> `EquipmentLabels`)

File: src/Fixtures/index.js [hunk 4]
```diff
@@ -1,11 +1,892 @@
 import animationMenageATrois from './AnimationMenageATrois';
 import animationSquare from './AnimationSquare';
 
+export const types = {
```
Comment (tonnenpinguin) [inline src/Fixtures/index.js:4]: types is a bit too generic.

File: src/Fixtures/index.js [hunk 5]
```diff
@@ -1,11 +1,892 @@
...
+};
+
+export const equipmentLabels = {
+  NOEQUIPMENT: 'No Equipment',
+  BASIC: 'Basic Equipment',
+  FULLEQUIPMENT: 'Full equipment',
+};
+
 export default {
+  favoritesDrill: [],
```
Comment (tonnenpinguin) [inline src/Fixtures/index.js:49]: `drillFavorites` or `favoriteDrills` might be a better name

File: src/Fixtures/index.js [hunk 6]
```diff
@@ -45,13 +926,13 @@ export default {
       equipment: 'One disc, 2 cones, 3 players',
       durationInMinutes: 10,
       goals: ['Defense', 'Handling'],
-      level: 'beginner',
+      level: 'Beginner',
```
Comment (tonnenpinguin) [inline src/Fixtures/index.js:929]: use enum

File: src/Fixtures/index.js (hunk 4 above)
Comment (sallesma) [reply src/Fixtures/index.js]: Maybe `categories` ?

File: src/Fixtures/index.js (hunk 5 above)
Comment (sallesma) [reply src/Fixtures/index.js:49]: `favoriteDrills` looks better to me

**Comment by guillaumemoby** [comment]
I ve apply the change that you have suggest but I can't commit anymore because of the test.

File: src/Fixtures/index.js (hunk 5 above)
Comment (sallesma) [reply src/Fixtures/index.js:52]: but we should remove it here, it should be added in the other pull request

**Comment by guillaumemoby** [comment]
@Martin est ce que je dois encore enlever " favoritesDrill: []," sachant que dans master il est aussi présent ?

**Pull request merged**